lib/marin/src/marin/post_training/flax/train.py

```diff
@@ -526,16 +525,10 @@ def main(config: TrainingConfig):
     jax_distributed_initalize(**config.distributed.jax_distributed_initialize_config)
     jax_distributed_barrier()
 
-    # Load attention kernel configurations
-    prefill_attention_kernel, prefill_attention_kernel_config = load_attention_kernel_config(
-        config.model.prefill_attention_kernel_config, ["splash", "default"]
-    )
-    generate_attention_kernel, generate_attention_kernel_config = load_attention_kernel_config(
-        config.model.generate_attention_kernel_config, ["paged", "default"]
-    )
-    train_attention_kernel, train_attention_kernel_config = load_attention_kernel_config(
-        config.model.train_attention_kernel_config, ["splash", "default", "ring", "ring_jax"]
-    )
+    # Validate attention kernel configurations (will raise ValueError if malformed)
+    load_attention_kernel_config(config.model.prefill_attention_kernel_config, ["splash", "default"])
```

Comment:
I wonder why these got reformatted now but not previously? Maybe we're using a slightly different formatter version now?